### PR TITLE
change configuration files path to debian defaults

### PR DESCRIPTION
**Pull request by EdoPut**
this will change all the configuration files path to the one that
ships with debian stretch. I added a note for users with other distro
or version.

Also fixed a sphinx build error with using ~ instead of ^ for
subsubparagraphs

### CODE REVIEW

File: docs/source/general/freeradius.rst [hunk 1]
```diff
@@ -239,7 +242,7 @@ and ``Calling-Station-ID``:
     echo $request | radclient localhost auth testing123
```
Comment (peppelinux) [inline docs/source/general/freeradius.rst]: radclient for acct and radtest for auth. 
In the comment we read about radtest but example show us a radclient command

Comment (EdoPut) [reply docs/source/general/freeradius.rst]: Thy are both shown with their relative command

**Comment by EdoPut** [comment]
> We should manage in [...] Debian stable
Debian stretch is the current stable

As for ubuntu there will be a LTS release early in 2018 and we can target that.

I don't understand how a path variable would make the docs better, they are a static page. Maybe we can put some footmarks with a reference to path in other distro.

For what is worth I'm adding a link to the freeradius wiki that explains a lot better that this article the configuration

File: docs/source/general/freeradius.rst [hunk 2]
```diff
@@ -83,15 +87,15 @@ Restart freeradius to load the new configuration:
 
 You may also want to take a look at the `Freeradius documentation <http://freeradius.org/doc/>`_.
 
-How to configure the REST module
---------------------------------
+Configure the REST module
+^^^^^^^^^^^^^^^^^^^^^^^^^
```
Comment (nemesisdesign) [inline docs/source/general/freeradius.rst]: why this?

Comment (nemesisdesign) [reply docs/source/general/freeradius.rst]: I mean the use of `^` instead of `-`

**Comment by EdoPut** [comment]
Because is a subsubsection of the subsection "Configure freeradius"

On Dec 31, 2017 19:02, "Federico Capoano" <notifications@github.com> wrote:

*@nemesisdesign* commented on this pull request.
------------------------------

In docs/source/general/freeradius.rst
<https://github.com/openwisp/django-freeradius/pull/84#discussion_r159144883>
:

> @@ -83,15 +87,15 @@ Restart freeradius to load the new configuration:

 You may also want to take a look at the `Freeradius documentation
<http://freeradius.org/doc/>`_.

-How to configure the REST module
---------------------------------
+Configure the REST module
+^^^^^^^^^^^^^^^^^^^^^^^^^

I mean the use of ^ instead of -

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/openwisp/django-freeradius/pull/84#discussion_r159144883>,
or mute the thread

**Pull request merged**